### PR TITLE
First try 4 real

### PR DESCRIPTION
**Pull request by kitpao**
We finally finished our first project, please review it :)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+                <h3>Published on Feb 27, 2014<br><br><br><br></h3>
+                  <p>Never miss a talk! SUBSCRIBE to the TEDx channel:
+                    <a href="http://bit.ly/1FAg8hB"> http://bit.ly/1FAg8hB</a><br><br>Josh Kaufman
+                    is the author of the #1 international bestseller, 'The Personal MBA: Master the Art of Business',
+                    as well as the upcoming book 'The First 20 Hours: Mastering the Toughest Part of
+                    Learning Anything.' </p>
+                  <h2><br><br>VIEW MORE</h2>
+              </div>
+              <div class="subscribe">
+                <a href="#"><img src="images/subscribe.png" alt="subsImg"></a>
```
Comment (Ispirett) [inline index.html]: use the `<button>` tag here instead of the `<a>` tag :+1:

**Review by Ispirett — APPROVED** [review]
**Hey, Folks  Excellent  job on your first project** 
image. 

**This project is complete you can merge.**

#### Before you merge please make the change requested below.

Good luck with your next project.

[ Optional ] not required to complete this project
It is a convention to use the `<main>` tag to wrap all dominant content in your HTML :+1:

File: index.html (hunk 1 above)
Comment (leonmezu1) [reply index.html]: Thank you very much !! 
We have doubts about the tag "1" because we don't find it.

Comment (leonmezu1) [reply index.html]: Do you mean we should replace the <a> tag with any other tag?

Comment (Ispirett) [reply index.html]: Snippet was not showing I edited it.

Comment (kitpao) [reply index.html]: thank you! it is done ^^

Comment (leonmezu1) [reply index.html]: Ok, the adjustments are done, could you review them please?

File: index.html [hunk 2]
```diff
@@ -40,56 +40,59 @@
     </header>
 
     <div class="content">
-      <section>
+      <main>
```
Comment (kitpao) [inline index.html:43]: here is the main (youtube video + comment section)

File: index.html [hunk 3]
```diff
@@ -40,56 +40,59 @@
...
+              <a href="http://bit.ly/1FAg8hB"> http://bit.ly/1FAg8hB</a><br><br>Josh Kaufman
+              is the author of the #1 international bestseller, 'The Personal MBA: Master the Art of Business',
+              as well as the upcoming book 'The First 20 Hours: Mastering the Toughest Part of
+              Learning Anything.' </p>
+              <h2><br><br>VIEW MORE</h2>
             </div>
+            <div class="subscribe">
+              <button type="button" name="subscribe_button" >
+                <img src="images/subscribe.png" alt="subsImg">
+              </button>
```
Comment (kitpao) [inline index.html:80]: Here is the new button

File: style.css [hunk 4]
```diff
@@ -187,9 +190,13 @@ section{
...
 }
 .subscribe img{
-  padding-left: 150px;
   height: 30px;
 }
+.subscribe button{
+  border-color: transparent;
+  height: 31px;
+  color: transparent;
+}
```
Comment (kitpao) [inline style.css:200]: css for button

**Review by Ispirett — APPROVED** [review]
Great job Folks Excellent job you can merge now

image. 

Interest branch name also  :+1:

**Pull request merged**